bindings_ffi/src/mls.rs

```diff
@@ -3146,6 +3147,47 @@ mod tests {
 
     use xmtp_cryptography::utils::generate_local_wallet;
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn radio_silence() {
+        let alex = new_test_client().await;
```

Comment:
itwould be nice to put this test closer to the code in `xmtp_api` or `xmtp_mls,` so it can run in wasm too